coral/src/app/features/configuration/environments/components/EnvironmentStatus.test.tsx

```diff
@@ -65,14 +65,22 @@ describe("EnvironmentStatus", () => {
       expect(button).toBeEnabled();
     });
   });
+
   describe("Refresh status", () => {
+    const originalConsoleError = console.error;
```

Comment:
Not blocking for this PR, but this is how I apply this pattern:

- Add the mock like you did
- in every `it` where I don't expect an errror, add a `expect(console.error).not.toHaveBeenCalled()`
- in the `it` block where the error will show up (and should), check with `expect(console.error).toHaveBeenCalledWith("error message")`

That way, we don't miss an unrelated `console.error` that could show up by swallowing it with the mock.

Reply:
Good point, thank you: [2b3136a](https://github.com/Aiven-Open/klaw/pull/1784/commits/2b3136a21f785fd5778edb9909bf58d8b8266d65)